src/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Movies App</title>
+</head>
+
+<body>
+
+    <header>
+        <div class="container">
+
+            <nav>
+                <div class="logo">
+                    <h1> Bahati & Antoine </h1>
+                </div>
+
+                <ul>
+                    <li>
+                        <span><a href="index.html" class="active">Browse</a>
+                        </span>
+                    </li>
+                    <li>
+                        <p class="total">Total Movies: <span id="total-movies">0</span></p>
+                        
+                    </li>
+                 
+                </ul>
```

Comment:
- [ ] Good job adding the Movies Counter to your Navigation. But kindly ensure your design matches the given wireframe in terms of the layout of elements on the page.


Your design
![image](https://github.com/ANTOINE1128/javascript-capstone/assets/81932478/1e23aaf0-34b3-4c10-a9c4-7a9bb15b6f96)

Give wireframe
![image](https://github.com/ANTOINE1128/javascript-capstone/assets/81932478/55cb624f-855d-4e30-983e-941249befab6)

#### Note the underline and the brackets around the count.